extensions/wikia/ContentReview/models/ReviewModel.php

```diff
@@ -192,4 +194,21 @@ public function getContentToReviewFromDatabase() {
 
 		return $content;
 	}
+
+	public function getRowToLogDatabase( $wiki_id, $page_id, $status ) {
```

Comment:
Please rethink this method name. It should describe what the code is doing, not for what we need it ;)